.gitignore

```diff
@@ -9,3 +9,4 @@ dist
 skepticoin/scmversion.py
 
 __pycache__
+*.code-workspace
```

Comment:
Nitpick: please keep developer-specific tastes out of this project's .gitignore and configure them on your local machine